tcf_website/templates/search/search.html

```diff
@@ -24,12 +24,21 @@ <h2>
           <div class="col-12">
             <div class="btn-group btn-group-sm" role="toolbar">
               <a class="btn bg-tcf-indigo text-white" disabled>See results for</a>
+              {% if courses_first %}
               <a href="#courses" id="course-tab" class="btn btn-light collapsed" autofocus
```

Comment:
My understanding is that the only difference here is which of the HTML tags has the `autofocus` attribute. If that's true, then you can simplify this logic to avoid duplicating the components themselves, like this:

```suggestion
              <a href="#courses" id="course-tab" class="btn btn-light collapsed" {% if courses_first %} autofocus {% endif %}
```

Note that if you want to see if `courses_first` is false, you'd do `{% if not courses_first %} {% endif %}`